pytissueoptics/scene/tree/treeConstructor/utils.py

```diff
@@ -14,3 +14,9 @@ def meanCentroid(axis: str, polygons: List[Polygon]):
             average += polygon.centroid.z
     average = average / len(polygons)
     return average
+
+def getPolygonsBbox(polygons: List[Polygon]):
+    bbox = BoundingBox([0, 0], [0, 0], [0, 0])
+    for polygon in polygons:
+        bbox.extendTo(polygon.bbox)
+    return bbox
```

Comment:
If these 2 private methods were moved out to utils, it's to test them, but they are currently not tested.

Reply:
Resolved with 4408122614e7a8e265f5b5f95ee0644c2040a1cd